tailwind.config.ts

```diff
@@ -14,7 +15,9 @@ module.exports = {
     },
     plugins: [
         animate,
-        reactAriaComponents({ prefix: 'rac' }),
```

Comment:
I'm not sure what this prefix: 'rac' argument did, but the type definition for the function says it does not exist. removing it doesnt seem to have changed anything.

Reply:
It was because of this issue: #39 
I checked on the preview of this PR and it doesn't happen.